.github/workflows/greetings.yml

```
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thanks for submitting your first issue to Wagtail!'
+        pr-message: 'Thanks for submitting your first PR to Wagtail!'
```

I think this is a great idea! I'd just like to suggest we give the PR submitters some reassurance that we will review their work as soon as we can.

```suggestion
        pr-message: 'Thanks for submitting your first PR to Wagtail! We know how hard you worked on this. Please give us some time to review, and we'll be in touch as soon as we can.'
```